homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -82,25 +166,7 @@ async def async_get_pipeline(
         pipeline_id = pipeline_data.pipeline_store.async_get_preferred_item()
 
     if pipeline_id is None:
-        # There's no preferred pipeline, construct a pipeline for the
-        # configured language
-        stt_engine = stt.async_default_provider(hass)
-        stt_language = hass.config.language if stt_engine else None
-        tts_engine = tts.async_default_engine(hass)
-        tts_language = hass.config.language if tts_engine else None
-        return await pipeline_data.pipeline_store.async_create_item(
-            {
-                "conversation_engine": None,
-                "conversation_language": None,
-                "language": hass.config.language,
-                "name": hass.config.language,
-                "stt_engine": stt_engine,
-                "stt_language": stt_language,
-                "tts_engine": tts_engine,
-                "tts_language": tts_language,
-                "tts_voice": None,
-            }
-        )
+        raise HomeAssistantError("no default assist pipeline")
```

Comment:
This is impossible, as we always create one and it becomes preferred.

Reply:
OK, modified so it's not possible to not have a preferred pipeline